docs/spec.md

```diff
@@ -3082,7 +3153,9 @@ Applies a map function `computation` to `inputs` along the `dimensions` and
 produces a `result` tensor.
 
 More formally, `result[i0, ..., iR-1] = computation(inputs[0][i0, ..., iR-1],`
-`..., inputs[N-1][i0, ..., iR-1])`.
+`..., inputs[N-1][i0, ..., iR-1])`. Note that `dimensions` are currently unused
+and will likely be removed in the future
+([#487](https://github.com/openxla/stablehlo/issues/487)).
```

Comment:
#487 not in frontend contract

Reply:
Same as [above](https://github.com/openxla/stablehlo/pull/1076#discussion_r1099053266).